src/main/java/appeng/block/misc/BlockSkyCompass.java

```diff
@@ -164,4 +186,31 @@ public void addCollidingBlockToList( final World w, final BlockPos pos, final Ax
 	{
 
 	}
+
+	@Override
+	public EnumBlockRenderType getRenderType( IBlockState state )
+	{
+		return EnumBlockRenderType.ENTITYBLOCK_ANIMATED;
+	}
+
+	@Override
+	@SideOnly( Side.CLIENT )
+	public TileEntitySpecialRenderer<? extends AEBaseTile> getTESR()
+	{
+		return new SkyCompassTESR();
+	}
+
+	@Override
+	@SideOnly( Side.CLIENT )
+	public ItemMeshDefinition getItemMeshDefinition()
+	{
+		return is -> new ModelResourceLocation( "appliedenergistics2:tile.BlockSkyCompass", "normal" );
```

Comment:
The source/target was a bit accidental as like 2 lines of java8 code somehow slipped into and but nobody ever wanted to fix it as there was no final decision about java8 or not.